src/App.js

```diff
@@ -1,12 +1,39 @@
 import './App.css';
+
+import QuestionPrompt from './components/QuestionPrompt';
+import { useState } from 'react';
 import Start from './components/Start';
 import startImage from './images/start_screen.png';
 
 function App() {
+  const [isInStart, setisInStart] = useState(true);
+  const [answered, setAnswered] = useState(-1);
```

Comment:
Instead of naming this variable `answered`, it is a little clearer as `questionsSeen` or something I think since it gets incremented on start before anything is actually answered

Reply:
This is working great though! 🌟